test/cmake/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.11 FATAL_ERROR)
+
+project(Likwid-Test LANGUAGES C)
+
+find_package(likwid REQUIRED COMPONENTS marker OPTIONAL_COMPONENTS nvmarker)
+
```

Comment:
The test doesn't compile since it doesn't find include path, add `include_directories(${LIKWID_INCLUDE_DIR})`

Reply:
Are you testing from the build tree? This test requires a make install

Reply:
Also this is modern CMake. You don't call include_directories, the include directory is automatically included when you link to "likwid::likwid" because I did `target_include_directories(likwid::likwid INTERFACE ${LIKWID_INCLUDE_DIRS})`

Reply:
If you follow the validation steps I laid out above, you'll see that it does indeed compile and the pre-processor flags are added